src/auth/auth.ts

```diff
@@ -411,19 +425,22 @@ export const loadOidcUser = (user: OidcUser): void => {
       // Load whether a user has input a cookie acceptance in a previous session on this system,
       // or whether they input cookie acceptance previously in this session
       cookiesAccepted: state.cookiesAccepted || getLocalPrefForUserId(userId, cookiesAcceptedKey),
-      terraUser: {
-        token: user.access_token,
-        scope: user.scope,
-        id: userId,
-        email: tokenClaims.email,
-        name: tokenClaims.name,
-        givenName: tokenClaims.given_name,
-        familyName: tokenClaims.family_name,
-        imageUrl: tokenClaims.picture,
-        idp: tokenClaims.idp,
-      },
     };
   });
+  userStore.update((state) => ({
+    ...state,
+    terraUser: {
+      token: user.access_token,
+      scope: user.scope,
+      id: userId,
+      email: tokenClaims.email,
+      name: tokenClaims.name,
```

Comment:
Is the fact that there's all three `name` `givenName` and `familyName` just an artifact of our user structure? Or is `name` a preferred name or something? It seems redundant to have all three

Reply:
The TerraUser object is a weird Frankenstein of the oidc claims. Ideally we would never use this object and instead only use the user profile. So the names are based on the mandatory claims which are in the access token. I think rather than try and fix this object, we should transition away from using it altogether.

Reply:
I agree, I don't think this object is useful at this point. It's redundant to have info duplicated in two user objects. As well as being confusing it doesn't get us anything we don't already have access to.

Reply:
Some of this stuff seems to be conflicting with the description of the purpose of each store. Based on the comments, it seems like information from token claims would go in OIDC store?